test/e2e/suite-ingress/suite-ingress-features/status.go

```diff
@@ -275,6 +276,20 @@ wrw7im4TNSAdwVX4Y1F4svJ2as5SJn5QYGAzXDixNuwzXYrpP9rzA2s=
 		assert.Contains(ginkgo.GinkgoT(), output, `status: "False"`, "status.conditions.status  is recorded")
 	})
 
+	ginkgo.It("check ApisixClusterConfig status is recorded", func() {
+		// create ApisixClusterConfig resource
+		clusterConfigName := config.ApisixV2
+		assert.Nil(ginkgo.GinkgoT(), s.NewApisixClusterConfig(clusterConfigName, true, true), "create cluster config error")
+		time.Sleep(6 * time.Second)
+
+		// status should be recorded as successfulen
+		output, err := s.GetOutputFromString("acc", clusterConfigName, "-o", "yaml")
+		assert.Nil(ginkgo.GinkgoT(), err, "Get output of ApisixPluginConfig resource")
+		assert.Contains(ginkgo.GinkgoT(), output, "type: ResourcesAvailable", "status.conditions.type is recorded")
```

Comment:
Did you check the error? It was due to
```
does not contain "type: ResourcesAvailable"
```
You can remove this assert.

Reply:
Yeah, that is a nice choice.